create-analysis-module.py

```diff
@@ -259,6 +260,40 @@ def main() -> None:
 
         final_messages.append(f"- Added Jupyter Notebook template: `{module_ipynb}`.")
 
+    # Add GHA workflows
+    workflow_template_dir = base_dir / "templates" / "workflows"
+    workflows_dir = base_dir / ".github" / "workflows"
+
+    # run module template (default to conda)
+    if args.use_r:
+        gha_template_source = "run_renv-module.yml"
+    else:
+        gha_template_source = "run_conda-module.yml"
+
+    gha_template = workflows_dir / f"run_{args.name}.yml"
+    copy_file_with_tag_replacement(
+        src=workflow_template_dir / gha_template_source,
+        dest=gha_template,
+        tag="openscpca_module",
+        replacement=args.name,
```

Comment:
```suggestion
        replacement=args.name
```

more generally though I've noticed that you often have commas at the end of a set of arguments into a function - is this intentional or otherwise strategic?